projects/canopy/src/lib/card/card.component.spec.ts

```diff
@@ -200,4 +202,28 @@ describe('LgCardComponent', () => {
       expect(component.cardToggableContent.isActive).toBe(false);
     });
   });
+
+  describe('when there is the lg-card-navigation-title', () => {
+    beforeEach(() => {
+      const localFixture = MockRender(`
+        <lg-card>
+         <lg-card-header>
+            <lg-card-navigation-title title="Greetings" link="https://www.landg.com" headingLevel="2"></lg-card-navigation-title>
+         </lg-card-header>
+         <lg-card-content>
+            Hello
+         </lg-card-content>
+        </lg-card>
+      `);
+
+      debugElement = localFixture.debugElement;
+      el = debugElement.children[0].nativeElement;
+      component = debugElement.children[0].componentInstance;
+      localFixture.detectChanges();
+    });
+
+    it('should set the class lg-card--navigation on the lg-card', () => {
```

Comment:
Maybe you could add a test outside this describe to say `'shouldn't set the class lg-card--navigation on the lg-card when the lg-card-navigation-title component is not present'`